test/expect/HasDecompTest.test_has_decomposition.expect

```diff
@@ -465,6 +465,7 @@ aten::_resize_output.out
 aten::_resize_output_
 aten::_sample_dirichlet
 aten::_sample_dirichlet.out
```

Comment:
might need to be some other updates, analogous to: https://github.com/pytorch/pytorch/pull/117390